.flake8

```diff
@@ -41,3 +44,4 @@ exclude =
     # This contains of branch that we don't want to check
     # dev
 max-complexity = 10
+max-line-length = 120
```

Comment:
I am nervous this will force a lot of unrelated changes for PRs that are fixing small issues. There are likely a lot of instances of line length > 120

Reply:
It is correct the project has many max-line-lenght problems, with this contribution https://github.com/DefectDojo/django-DefectDojo/pull/8046
we incorporate a pre-commit so that the technical debt does not continue to grow.

In the future we should remove all the ignore in the .flacke8 file.

Reply:
It would be ideal to correct all instances of line length violations before enabling this rule. Otherwise, all PRs going forwards will have failing unit tests until the entire project is no longer in violation

Reply:
@maffooch and @mtesauro, thank you for your contributions.

As far as I understand, you are running flake8 in the pipeline, but ignoring several errors. At this point, we agree not to modify these rules until we have everything corrected. Not only the maximum number of lines, which is the easiest and can be achieved with black automatically up to a certain point and is already integrated in the PR pre-commit. https://github.com/DefectDojo/django-DefectDojo/pull/8053

Initially, we thought about creating a single PR https://github.com/DefectDojo/django-DefectDojo/pull/8013 with all the corrections and improvements in code readability according to flake8, black, pep8. But we realized that there may be too many changes and it could complicate things for merging.

That's why we launched this PR with the necessary corrections reported by SonarQube for the dojo/finding/views.py module, making it much easier to review and reduce the chances of conflicts with other PRs.

The idea is to continue contributing with small contributions module by module incrementally until technical debt is completely eliminated. But before that, we must avoid it from growing, which is why we propose this pre-commit. It will only run on the modules that were modified and not on the entire project, and it is the developer's responsibility to correct it if necessary. At this stage, it is important to remember that we have black, which saves us a lot of work. I say this because I have already done it, and it didn't take me more than two days to correct the entire project with the proposed pre-commit configuration.

@mtesauro, regarding your recommendation, we totally agree. We are not taking into account (./test) and (./unittest).

In this image, we can see the problems we currently have. Excluding the coverage.

![image](https://user-images.githubusercontent.com/40049733/236311588-04fe2d3e-2087-4e09-9ad8-58da46bc9acf.png)

Reply:
@renejal I think we are on the same page now. I think the course of action should be as follows
1. Close #8013 
2. Update the code base one by one to be in compliance with Black, Pep8, and Flake8
3. Once the code base is in compliance, add the pre commit hooks and update the unit tests to ensure the code base stays in compliance.

Reply:
Perfect